README.md

```diff
@@ -30,11 +39,116 @@ timdex_dataset_api = {git = "https://github.com/MITLibraries/timdex-dataset-api.
 
 ### Required
 
+None at this time.
+
 ### Optional
 ```shell
 TDA_LOG_LEVEL=# log level for timdex-dataset-api, accepts [DEBUG, INFO, WARNING, ERROR], default INFO
+WARNING_ONLY_LOGGERS=# Comma-seperated list of logger names to set as WARNING only, e.g. 'botocore,charset_normalizer,smart_open'
 ```
 
 ## Usage
 
-_TODO..._
```

Comment:
These README updates were just a bit of retroactive TODO cleanup.  It's not super detailed, and does not include any mention of `TIMDEXDatasetMetadata` functionality, but hopefully demonstrates a bit how the library is used.

FWIW, I might prefer to make any further updates or additions to this section as a separate ticket/PR to keep this one focused on `TIMDEXDatasetMetadata`.